### PR TITLE
Update gems and add bin/rspec.

### PR DESCRIPTION
**Pull request by squaresurf**
- https://github.com/MiniProfiler/rack-mini-profiler
- https://github.com/schneems/derailed_benchmarks
- https://github.com/tmm1/stackprof
- bin/rspec in order to take advantage of spring when running tests.

- to rails 4.2.5 from 4.2.4.
- shoulda-matchers to only be in the test gem group

### CODE REVIEW

**Comment by Mrjaco12** [comment]
:+1:

**Comment by bbrock25** [comment]
:+1:

File: Gemfile [hunk 1]
```diff
@@ -38,12 +39,15 @@ group :development, :test do
...
   gem 'pry-remote'
+  gem 'rack-mini-profiler'
   gem 'rspec-rails'
-  gem 'shoulda-matchers'
+  gem 'stackprof'
 end
 
 group :development do
   gem 'refills'
+  gem 'shoulda-matchers'
```
Comment (samcdavid) [inline Gemfile]: Are we sure `'shoulda-matchers'` doesn't have to be in the `:test` environment?

Comment (squaresurf) [inline Gemfile]: Great catch @samcdavid!!! I meant to move this into the `test` group as it is **not** needed in `development`.

**Comment by squaresurf** [comment]
@samcdavid would you please take another look?

**Comment by samcdavid** [comment]
Sorry for the delay, the octocat notifier is not notifying me for some reason :frowning: 

:+1:

**Pull request merged**